co-log/co-log.cabal

```
@@ -86,7 +86,7 @@ library
                            Colog.Pure
                            Colog.Rotation
 
-  build-depends:       ansi-terminal ^>= 0.10
+  build-depends:       ansi-terminal ^>= 0.11
```

I would keep lower bound as well

```suggestion
  build-depends:       ansi-terminal >= 0.10 && < 0.12
```

---

Thanks, updated.